packages/perspective/package.json

```diff
@@ -24,13 +24,17 @@
         "bench": "npm-run-all bench:build bench:run",
         "prebuild": "mkdir -p build && CLICOLOR_FORCE=1 yarn compile",
         "precompile": "mkdir -p obj && mkdir -p ../../obj/",
-        "compile": "cd ../../obj/ && emcmake cmake ../scripts/ && emmake make -j${PSP_CPU_COUNT-8}",
+        "compile": "cd ../../obj/ && emcmake cmake ../ && emmake make -j${PSP_CPU_COUNT-8}",
```

Comment:
IMO this stuff should all just be in the TLD for the project.

Reply:
This build is a dependency of `@jpmorganchase/perspective`, so as a developer, I'd like it only run when I build this package, not e.g. when I build `@jpmorganchase/perspective-jupyterlab`.